src/fp6.rs

```diff
@@ -110,6 +110,13 @@ impl Fp6 {
         }
     }
 
+    #[inline]
+    // #[cfg(target_os = "zkvm")]
+    pub fn mul_inp(&mut self, rhs: &Fp6) {
+        // TODO
```

Comment:
For context of others reading this and wondering why the `TODO`: the [`Fp6::mul` function](https://github.com/lurk-lab/bls12_381/blob/5eb990030dca1d5b971bca87b6d5086e856dcfc9/src/fp6.rs#L200-L274) is quite involved, and it's not clear that there's anything we can further optimize here necessarily, since `Fp::sum_of_products` is already optimized.

However, it's also a very important function in the hot path called many times, so it's possibly still worth taking a look in the future and seeing if we can prevent a copy by reusing `&self` when calculating the .`Fp::sum_of_products` stuff. This `TODO` is more about doing this investigation and figuring out if it's worth it

Reply:
I will add your comment in the TODO